examples/remix-zip/README.md

```diff
@@ -0,0 +1,37 @@
+# remix-zip
+
+This example shows how to use Lambda Adapter to run an Remix application on AWS Lambda. 
+
+The Remix application is created using the following command: 
+
+```bash
+npx create-remix@latest --template remix-run/remix/templates/express
+```
+
+### How does it work?
+
+We add Lambda Adapter layer to the function and configure wrapper script. 
+
+1. attach Lambda Adapter layer to your function. This layer containers Lambda Adapter binary and a wrapper script. 
+    1. x86_64: `arn:aws:lambda:${AWS::Region}:753240598075:layer:LambdaAdapterLayerX86:23`
+    2. arm64: `arn:aws:lambda:${AWS::Region}:753240598075:layer:LambdaAdapterLayerArm64:23`
+2. configure Lambda environment variable `AWS_LAMBDA_EXEC_WRAPPER` to `/opt/bootstrap`. This is a wrapper script included in the layer.
+3. set function handler to a startup command: `run.sh`. The wrapper script will execute this command to boot up your application. 
+
+To get more information of Wrapper script, please read Lambda documentation [here](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-modify.html#runtime-wrapper). 
+
+### Build and Deploy
+
+Run the following commands to build and deploy the application to lambda. SAM CLI will use GNU Make to build the Remix application.
+
+```bash
+sam build
+sam deploy --guided
+```
+When the deployment completes, take note of RemixFunctionApi's Value. It is the API Gateway endpoint URL. 
+
```

Comment:
Most of the other examples (and the Remix-image example below) have a section in the README for local testing. You should add something here related to that. Like [SpringBoot](https://github.com/awslabs/aws-lambda-web-adapter/tree/main/examples/springboot-zip#local-test-with-sam-cli), using SAM CLI, or if Remix has its own way for local testing.

Reply:
Sure, added the section.

Reply:
You added a SAM local testing section to the README inside the image app (which also had a way to locally test with docker directly), but didn't add it in this file (-zip version)

Reply:
Updated